velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -120,6 +120,39 @@ class DateTimeFunctionsTest : public functions::test::FunctionBaseTest {
         rowVector->children()[1]->as<SimpleVector<int16_t>>()->valueAt(0)};
   }
 
+  std::optional<TimestampWithTimezone> timestampWithTimezoneAtTimezone(
+      std::optional<int64_t> timestamp,
+      const std::optional<std::string>& timeZoneName,
```

Comment:
When can the arguments be optional?

Reply:
I agree, but I made them as such to match the signatures of the other functions I call, like `makeRowVector()` and `makeTimestampWithTimeZoneVector()`.

Reply:
True, added tests for null input.